openupgrade_scripts/scripts/website_slides_survey/16.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_translations_to_delete = [
+    "mail_template_user_input_certification_failed",
+]
+
+
+def _delete_sql_constraints(env):
+    # Delete constraints to recreate it
```

Comment:
It can be a docstring.

```suggestion
    """Delete constraints to recreate it"""
```